javalin/src/main/java/io/javalin/http/servlet/JavalinServletContext.kt

```diff
@@ -141,7 +144,12 @@ class JavalinServletContext(
         userFutureSupplier = future
     }
 
-    override fun skipRemainingHandlers(): Context = also { tasks.clear() }
+    override fun skipRemainingHandlers(): Context = also {
+        tasks.clear()
+        if (handlerType() == WEBSOCKET_BEFORE_UPGRADE) {
+            throw UnauthorizedResponse()
+        }
+    }
```

Comment:
Any concerns here @dzikoysk ?

Reply:
I don't really like bringing more dependency on websockets in Javalin internals - ideally, websockets should be treated as a standalone module.

Reply:
I don't necessarily disagree, but I don't think this is the PR to address this.